e2e_test/source_inline/tvf/postgres_query.slt

```diff
@@ -1,5 +1,13 @@
+control substitution on
+
+system ok
+source .risingwave/config/risedev-env
+
+system ok
+PGHOST=$PGHOST PGPORT=$PGPORT PGUSER=$PGUSER PGPASSWORD=postgres PGDATABASE=source_test createdb
```

Comment:
We create a new db, in case we are using the meta pg backend.

Reply:
This might be enough. `PGHOST=$PGHOST` isn't very meaningful 🤣
```suggestion
PGDATABASE=source_test createdb
```